var/spack/repos/builtin/packages/arrow/package.py

```diff
@@ -65,6 +67,8 @@ def cmake_args(self):
             args.append("-DARROW_PYTHON:BOOL=ON")
         if self.spec.satisfies('+parquet'):
             args.append("-DARROW_PARQUET:BOOL=ON")
+        if self.spec.satisfies('+orc'):
+            args.append('-DARROW_ORC:BOOL=ON')
```

Comment:
Is it possible to explicitly disable these in an else clause? CMake likes to switch defaults based on what it finds on your system, leading to non-deterministic builds.